phpstan-baseline.neon

```diff
@@ -880,6 +875,11 @@ parameters:
 			count: 1
 			path: src/lib/Service/ContentService.php
 
+		-
+			message: "#^Method EzSystems\\\\EzRecommendationClient\\\\Service\\\\ContentService\\:\\:prepareFields\\(\\) should return array\\<string\\> but returns array\\<int\\<0, max\\>, mixed\\>\\|null\\.$#"
+			count: 1
+			path: src/lib/Service/ContentService.php
+
```

Comment:
We should fix this in code.

Declaring an fields array before foreach (https://github.com/ezsystems/ezrecommendation-client/pull/115/files#diff-87b11ded75f12e5544670cd808d128fb771be122a87b40b3186092fbf5a10e9fR268) should help:

`$fields = [];`

Reply:
done